shell/platform/darwin/ios/ios_surface_metal_impeller.mm

```diff
@@ -4,10 +4,75 @@
 
 #import "flutter/shell/platform/darwin/ios/ios_surface_metal_impeller.h"
 
+#include "flutter/impeller/renderer/backend/metal/formats_mtl.h"
+#include "flutter/impeller/renderer/context.h"
 #include "flutter/shell/gpu/gpu_surface_metal_impeller.h"
 
+namespace impeller {
+namespace {
+using std::shared_ptr;
```

Comment:
Nit: Remove this?

Reply:
Yep, done.  I'm still mystified how this got here.  I wonder if my ide automatically added it.